test/ClangImporter/pcm-emit-direct-cc1-mode.swift

```diff
@@ -0,0 +1,16 @@
+// Emit the explicit module.
+// RUN: %empty-directory(%t)
+// RUN: %swift-frontend -emit-pcm -direct-clang-cc1-module-build -only-use-extra-clang-opts -module-name script -o %t/script.pcm %S/Inputs/custom-modules/module.map -Xcc -o -Xcc %t/script.pcm -Xcc -fmodules -Xcc -triple -Xcc %target-triple -Xcc objective-c -dump-clang-diagnostics 2> %t.diags.txt
```

Comment:
The Clang frontend command line is essentially `-o %t/script.pcm -fmodules -triple %target-triple objective-c`, correct? I'd a bit surprised if Clang doesn't complain about file `objective-c` not existing, since the flag is not prefixed with `-x`.

Reply:
Fixed, thanks for catching this.